chapter_09_docker.asciidoc

```diff
@@ -388,6 +388,10 @@ TIP: I'm deliberately choosing a different port to run Dockerised Django on (888
 
 I'll use the `--failfast` option to exit as soon as a single test fails:
 
+// CSANAD:  are line breaks necessary below and at other occurrences of
+//          --failfast? it does keep the asciidoc lines of source below 80
+// characters but there are some wider lines elsewhere.
```

Comment:
by default, source code has to break around 80 chars, otherwise it won't fit on the printed page.

there is a `role=small-code` directive we can use when necessary, but i try and avoid it when i can...